build.gradle

```diff
@@ -184,6 +185,12 @@ integTest {
         }
     }
 
+    if (System.getProperty("tests.rest.suite") == null) {
+        filter {
```

Comment:
Looks like the tests are explicitly excluded in integTests, if BWC tests do not have any dependencies we should just let it run as part of the integtests which will be part of `gradle build` and will run in CI for every PR commit.

Ref: AD CI https://github.com/opensearch-project/anomaly-detection/blob/main/.github/workflows/CI.yml

Reply:
They are currently excluded from the `integTest` as they need some additional setup to be run in CI, if I include them with `integTest` right now without the changes, the workflow would fail. I figured that we could exclude currently and then hook them to CI (as part of `integTest`) as part of the next task. LMK what you think.